source/features/bugs-tab.tsx

```diff
@@ -24,14 +24,46 @@ async function highlightBugsTabOnIssuePage(): Promise<void | false> {
 	issuesTab.removeAttribute('aria-current');
 }
 
+const bugLabelCacheKey = (): string => 'bugs-label:' + getRepo()!.nameWithOwner;
+const bugLabel = async (): Promise<string | undefined> => cache.get<string>(bugLabelCacheKey());
+
 const countBugs = cache.function(async (): Promise<number> => {
-	const {search} = await api.v4(`
-		search(type: ISSUE, query: "label:bug is:open is:issue repo:${getRepo()!.nameWithOwner}") {
-			issueCount
+	if (!await bugLabel()) {
```

Comment:
No?

```suggestion
	const label = cache.get<string>(bugLabelCacheKey());
	if (!label) {
```

Reply:
I need it for the next function

Reply:
Not sure I follow. From what I see, if it doesn't exist in the cache, we enter a block that sets both the bug count and bug label into the cache and then it's done. 

If it *does* exist, it's just used as part of the label-specific API call. 

I also don't get when it could be "not ready" as you mentioned in the other comment

Reply:
https://github.com/sindresorhus/refined-github/blob/517cc5d0e27d9b7587f01aa7a26fd871daf49c62/source/features/bugs-tab.tsx#L77-L82

This line runs as soon as the page loads, but before `countPromise` is done.

Also that is the line that I need it again

Reply:
```suggestion
	const label = await bugLabel();
	if (!label) {
```